File: app/src/main/java/com/example/learningtasks/java/BankAccount.java
package com.example.learningtasks.java;



public class BankAccount {
    private double balance;


    public BankAccount(double balance) {
        this.balance = balance;
    }

    public void deposit(double amount) {
        if (amount > 0) {
            balance += amount;
        }
    }

    public void withdraw(double amount) {
        if (amount > 0 && amount < balance) {
            balance -= amount;
        }
    }

    public double getBalance() {
        return balance;
    }


    public static void main(String[] args) {
        BankAccount account = new BankAccount(1000.0);

        account.deposit(300.0);
        System.out.println("Deposited Amount = " + account.getBalance());
        account.withdraw(400.0);
        System.out.println("Withdrawal Amount = " + account.getBalance());
        System.out.println("Your Balance = " + account.getBalance());
    }
}

File: app/src/main/java/com/example/learningtasks/BankAccount.kt
package com.example.learningtasks


class BankAccount(private var Balance: Double) {

    fun deposit(amount: Double) {
        if (amount > 0) {
            Balance += amount
        }
    }

    fun withdraw(amount: Double) {
        if(amount > 0 && amount < Balance) {
            Balance -= amount

        }
    }

    fun getBalance(): Double {
        return Balance
    }
}


fun main() {
    val account = BankAccount(Balance = 1000.0)

    account.deposit(300.0)
    println("Deposited Amount = ${account.getBalance()}")
    account.withdraw(400.0)
    println("withdrawal Amount = ${account.getBalance()}")
    account.getBalance()
    println("Your Balance = ${account.getBalance()}")
}
